setup.py

```diff
@@ -0,0 +1,8 @@
+from setuptools import setup, find_packages
+setup(
+    name = "pystorage",
```

Comment:
Should it be this name?

Reply:
No, miss copy :smile: . Updated @vavieira10